### PR TITLE
[KOGITO-413] -bump kogito images version and publish it on quay.io

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: Filippe Spolti <fspolti@redhat.com>

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]

File: s2i/tests/features/kogito-quarkus-index.feature [hunk 1]
```diff
@@ -14,14 +14,13 @@ Feature: Kogito-quarkus-ubi8 feature.
...
   Scenario: verify if the binary index is available on /home/kogito
     When container is started with command bash
-    Then run sh -c 'ls /home/kogito/bin/data-index-service-*-runner.jar' in container and immediately check its output for /home/kogito/bin/data-index-service-0.4.0-runner.jar
+    Then run sh -c 'ls /home/kogito/bin/data-index-service-*-runner.jar' in container and immediately check its output for /home/kogito/bin/data-index-service-0.5.0-runner.jar
 
   Scenario: Verify data-index default configuration
     When container is started with env
       | variable     | value |
       | SCRIPT_DEBUG | true  |
-    Then container log should contain Listening on: http://0.0.0.0:8180
```
Comment (sutaakar) [inline s2i/tests/features/kogito-quarkus-index.feature:23]: Is this line removed because index service doesn't show this line any more?

Comment (spolti) [reply s2i/tests/features/kogito-quarkus-index.feature:23]: no, the index image does not correctly starts without infinispan thus this line is not printed.

Comment (ricardozanini) [reply s2i/tests/features/kogito-quarkus-index.feature:23]: ^ is **not** printed, you meant?

Comment (spolti) [reply s2i/tests/features/kogito-quarkus-index.feature:23]: yeah, typo, hehe

**Pull request merged**